### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by ahmed-ismaeil**
- added favicon
- modified styles
- more components
- rebuild text messages
- rebuild messages with a button
- updated json file with more data

### CODE REVIEW

File: src/Components/ChatRoom/index.js [hunk 1]
```diff
@@ -1,60 +1,29 @@
...
-        <MyMessag />
-        <OtherMessage />
-        <Date>
-          <DateP>Apr 1</DateP>
-        </Date>
+        <MsgContent />
+        
+        <ChatRoomDate />
+
         <MyBigMsg />
```
Comment (MarcMagdi) [inline src/Components/ChatRoom/index.js]: Shouldn't MyBigMsg appear first in the order as it's before the two other messages?

**Review by MarcMagdi — CHANGES_REQUESTED** [review]
Can we refactor the components to use hooks? 
The two hooks you'll mainly need is useEffect & useState

**Review by MarcMagdi — APPROVED** [review]

**Comment by MarcMagdi** [comment]
@ahmed-ismaeil Great Job :1st_place_medal:

**Comment by ahmed-ismaeil** [comment]
> @ahmed-ismaeil Great Job 🥇

@MarcMagdi Thank you

**Pull request merged**